Modules/hmacmodule.c

```diff
@@ -0,0 +1,277 @@
+#ifndef Py_BUILD_CORE_BUILTIN
+#  define Py_BUILD_CORE_MODULE 1
+#endif
+
+#include "pyconfig.h"
+#include "Python.h"
+#include "hashlib.h"
+
+#include "_hacl/Hacl_HMAC.h"
+
+#include "clinic/hmacmodule.c.h"
+
+#define HACL_HMAC_COMPUTE_HASH(HACL_FUNCTION, DIGEST_SIZE, KEY, SRC)    \
```

Comment:
I would enjoy a comment here explaining what this does, because it's used by basically everything in the file.